.github/workflows/release.yml

```
@@ -78,7 +78,7 @@ jobs:
     strategy:
       matrix:
         script:
-          - 'integration:ng15'
+          - 'integration:ng16'
```

Should we add `- 'integration:ng17'` here too?
```suggestion
          - 'integration:ng16'
          - 'integration:ng17'
```